config/software/berkshelf.rb

```diff
@@ -43,4 +43,18 @@
 
   gem "install pkg/berkshelf-*.gem" \
       " --no-ri --no-rdoc", env: env
+
+  if windows?
```

Comment:
still think this could safely be `unless aix?` since i think the `*.a` files are also worthless on the rest of the *nixes